internal/cephfs/errors.go

```diff
@@ -25,6 +28,30 @@ var (
 	// ErrNonStaticVolume is returned when a volume is detected as not being
 	// statically provisioned.
 	ErrNonStaticVolume = errors.New("volume not static")
+	// ErrVolNotEmpty is returned when a subvolume has snapshots in it.
+	ErrVolNotEmpty = fmt.Errorf("%v", "Error ENOTEMPTY")
```

Comment:
Just use `errors.New()` like done for `ErrNonStaticVolume` above

Reply:
@nixpanic  https://github.com/ceph/ceph-csi/pull/394#discussion_r462779851 :)